src/components/cylc/log/Log.vue

```diff
@@ -47,14 +66,29 @@ export default {
       required: false,
       default: false
     },
+    error: {
+      type: String,
+      required: false
+    },
+    autoScroll: {
+      type: Boolean,
+      required: false,
+      default: false
+    }
   },
 
   data () {
     return {
-      match: ''
+      match: '',
+      ro: null
     }
   },
 
+  mounted () {
+    this.ro = new ResizeObserver(this.onResize)
+    this.ro.observe(this.$el)
+  },
```

Comment:
This does not need to be reactive
```suggestion
    }
  },

  mounted () {
    const ro = new ResizeObserver(this.onResize)
    ro.observe(this.$el)
  },
```

Reply:
fixed in 49544afb8128260fe05f5020e87ca8767e666521